browser/ui/views/sidebar/sidebar_item_add_button.cc

```diff
@@ -50,6 +52,11 @@ void SidebarItemAddButton::OnMouseExited(const ui::MouseEvent& event) {
   SidebarButtonView::OnMouseExited(event);
   // Don't show bubble if user goes outo from add item quickly.
   timer_.Stop();
+
+  if (!add_item_bubble_shown_by_click_ && IsBubbleVisible()) {
```

Comment:
Suggestion: how about add one more condition and reorder them (as `add_item_bubble_shown_by_click_` is false most of the time? 

```suggestion
  if (!IsBubbleVisible() && !add_item_bubble_shown_by_click_ && 
 !add_item_bubble_widget_->IsClosed()) {
```

Reply:
And, I think we should check if the mouse moves onto the bubble. Otherwise, users can't use the bubble when it's triggered by hovering.

Reply:
It's intended behavior. design team want to use dialog as a tooltip.
With this PR, user should click that button to use dialog